browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -95,6 +96,24 @@ class MarkdownNoteDetail extends React.Component {
     return title
   }
 
+  getPercentageOfCompleteTodo (value) {
+    let splitted = value.split('\n')
+    let numberOfTodo = 0
+    let numberOfCompletedTodo = 0
+
+    for (let i = 0; i < splitted.length; i++) {
```

Comment:
Thank you for commenting!  Surely, It's long to using 'for' than using 'foreach'. So, I'll fix it.